packages/client/src/components/mfm.ts

```diff
@@ -91,7 +91,8 @@ export default defineComponent({
 					let style;
 					switch (token.props.name) {
 						case 'tada': {
-							style = `font-size: 150%;` + (this.$store.state.animatedMfm ? 'animation: tada 1s linear infinite both;' : '');
+							const speed = validTime(token.props.args.speed) || '1s';
+							style = `font-size: 150%;` + (this.$store.state.animatedMfm ? `animation: tada ${speed} linear infinite both;` : '');
```

Comment:
template string is not necessary for a constant string
```suggestion
							style = 'font-size: 150%;' + (this.$store.state.animatedMfm ? `animation: tada ${speed} linear infinite both;` : '');
```

Reply:
I overlooked it! Thank you!